docs/products/postgresql/concepts/timescaledb.rst

```diff
@@ -0,0 +1,31 @@
+About TimescaleDB
+=================
+
+TimescaleDB is an open-source database designed to make your existing relational database scalable for time series data. `TimescaleDB <https://github.com/timescale/timescaledb>`_ is available as a PostgreSQL extension on Aiven.
+
+A time series indexes a series of data points in chronological order, usually as a sequence over regular intervals. Examples of a time series include:
+
+* The temperature of a home during a day
```

Comment:
I'd suggest starting these bullet points with lower case as these are phrases and not complete sentences.

Reply:
updated